StickKnightsOnline-Client/main.cpp

```diff
@@ -8204,12 +8253,50 @@ void physics()
 
 
 
+	  //Sign reading collision detection
+	  if (enableSIGN) {
```

Comment:
Don't do this calculation in Graphics, do it in the Physics thread. Also, don't trigger it every loop when standing still, just when moving.